setup/01-manual_testing.py

```diff
@@ -26,10 +28,14 @@ def check_content():
             is_username = True
         elif "password" in lowered_line:
             is_password = True
-    if is_open_browser and is_url and is_password and is_username:
+        elif "click" in lowered_line or "login" in lowered_line:
+            is_login = True
+        elif "welcome" in lowered_line:
+            is_welcome = True
+    if is_open_browser and is_url and is_password and is_username and is_login and is_welcome:
         print("Ready to proceed!")
     else:
-        print("Not quite there yet! Did remember browser, username, password or perhaps missing url?")
+        print("Not quite there yet! Did remember browser, username, password or perhaps missing url? Do you verify your login?")
```

Comment:
```suggestion
        print("Not quite there yet! Did you remember browser, username, password or perhaps missing url? Did you verify your login?")
```
I don't understand what "did you verify your login" means?

Reply:
It means previously the exercise only validated that the participant had implemented `Enter Username` and `Enter Password`, but not verifying the actual login (that we reach Welcome page).